examples/with-docker/README.md

```diff
@@ -22,8 +22,6 @@ You can view your images created with `docker images`.
 
 ## Deploying to Google Cloud Run
 
-The `start` script in `package.json` has been modified to accept a `PORT` environment variable (for compatibility with Google Cloud Run).
-
```

Comment:
I believe it is not necessary to change the usage of `next start` since it will pick up `$PORT` value anyway (the `-p` flag is redundant if `$PORT` variable is present)

Reply:
This is explicitly needed for deployment to Google Cloud. Maybe we can add a comment?

Reply:
So I understand correctly - which part is needed for GC, the start script in package.json, or the `-p` parameter on next CLI, or both?

We don't pass the `-p` parameter in Dockerfile CMD since https://github.com/vercel/next.js/commit/5abfea3765c0d3fab87faf8e2ffacbafcacbb845.

Happy to close this PR if not needed!